------------------------------------------------------------
revno: 14157
revision-id: squid3@treenet.co.nz-20170529042116-kp9naxxmdsqicpjv
parent: squid3@treenet.co.nz-20170508110920-73gma737u4x6ce87
fixes bug: http://bugs.squid-cache.org/show_bug.cgi?id=4589
committer: Amos Jeffries <squid3@treenet.co.nz>
branch nick: 3.5
timestamp: Mon 2017-05-29 16:21:16 +1200
message:
  Bug 4589: ssl_crtd: returning zero on failure
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squid3@treenet.co.nz-20170529042116-kp9naxxmdsqicpjv
# target_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# testament_sha1: ad29dd184416dc47dee80234c541185cca166bb3
# timestamp: 2017-05-29 04:39:57 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# base_revision_id: squid3@treenet.co.nz-20170508110920-\
#   73gma737u4x6ce87
# 
# Begin patch
=== modified file 'src/ssl/ssl_crtd.cc'
--- src/ssl/ssl_crtd.cc	2017-01-01 00:16:45 +0000
+++ src/ssl/ssl_crtd.cc	2017-05-29 04:21:16 +0000
@@ -350,7 +350,7 @@
         }
     } catch (std::runtime_error & error) {
         std::cerr << argv[0] << ": " << error.what() << std::endl;
-        return 0;
+        return -1;
     }
     return 0;
 }

